core/domain/collection_domain.py

```diff
@@ -151,71 +147,6 @@ def __init__(self, change_dict):
             raise Exception('Invalid change_dict: %s' % change_dict)
 
 
-class CollectionCommitLogEntry(object):
```

Comment:
Are we no longer tracking these? Also, what are we going to do with the existing `CollectionCommitLogEntry` objects in our data store (if there are any)?

Reply:
/cc @seanlip

Reply:
This might be fine since we don't use the domain object. I think the storage model is put-only at the moment so I think it's probably fine.

That said, @lilithxxx, deletions need to be explained, especially major ones. Next time please leave comments in the description or inline in the code, for the reviewer. Thanks!

Reply:
Noted!

Reply:
@lilithxxx, can you note that in the PR description?

Reply:
Done

Reply:
thanks!